integration/monai/examples/spleen_ct_segmentation/virtualenv/requirements.txt

```diff
@@ -1,8 +1,8 @@
-tensorboard
 tqdm
 nibabel
 fire
 pytorch-ignite>=0.4.10
 monai>=1.0.1rc3
```

Comment:
this will cause an install error with monai_nvflare 0.2.1rc1. We need to build a new monai_nvflare first....
```
The conflict is caused by:
    The user requested monai>=1.0.1rc4
    monai-nvflare 0.2.1rc1 depends on monai==1.0.1rc3
```